### PR TITLE
bump netty* -> 4.1.73

### PR DESCRIPTION
**Pull request by snesm**
# BACKEND PULL REQUEST
Update netty* dependencies from 4.1.55.Final -> 4.1.73.Final

4.1.73.Final is the version being request by most parent dependencies.

## Related Issue
PR 3797

- Why is this being done? Link to issue, or a few sentences describing why this PR exists
Eleven Snyk vulnerabilities

## Changes Proposed

- Update to fix vulnerabilities.

## Additional Information

- decisions that were made
- notice of future work that needs to be done

## Testing

- How should reviewers verify this PR?

## Checklist for Primary Reviewer

- [x] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke tested
- [x] Any content updates (user-facing error messages, etc) have been approved by content team
- [x] Any changes that might generate questions in the support inbox have been flagged to the support team
- [x] GraphQL schema changes are backward compatible with older version of the front-end
- [ ] Changes comply with the SimpleReport Style Guide
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

### CODE REVIEW

**Comment by rin-skylight** [comment]
Successfully smoke tested on `pentest`!

**Review by rin-skylight — APPROVED** [review]
LGTM!

**Comment by snesm** [comment]
This may eventually be superseded by https://github.com/CDCgov/prime-simplereport/pull/3562

File: backend/build.gradle [hunk 1]
```diff
@@ -108,7 +126,6 @@ dependencies {
     runtimeOnly 'com.azure:azure-core-http-netty:1.11.9'
     runtimeOnly 'com.azure:azure-core:1.27.0'
     runtimeOnly 'com.azure:azure-storage-common:12.15.1'
-    runtimeOnly "com.azure:azure-storage-queue:12.12.1"
```
Comment (zdeveloper) [inline backend/build.gradle:111]: why is this dependency being removed? I know we still use the azure storage queue

File: backend/build.gradle [hunk 2]
```diff
@@ -79,6 +79,24 @@ dependencies {
 
     implementation 'com.azure:azure-storage-queue:12.12.+'
 
+    implementation 'io.netty:netty-codec:4.1.73.Final'
```
Comment (zdeveloper) [inline backend/build.gradle:82]: is this managed internally by Spring now? or are these new dependencies?

Comment (snesm) [reply backend/build.gradle:82]: Spring manages these.  All of this is unnecessary when https://github.com/CDCgov/prime-simplereport/pull/3562 is complete.  Until then, forcing these versions fixes 11 vulnerabilities.

**Review by nathancrtr — APPROVED** [review]
Assuming that @rin-skylight's [original assessment](https://github.com/CDCgov/prime-simplereport/pull/3798#issuecomment-1136471808) still stands this LGTM 👍

**Review by alismx — APPROVED** [review]
LGTM but I'm going to echo @nathancrtr  https://github.com/CDCgov/prime-simplereport/pull/3798#pullrequestreview-1030756445

**Review by rin-skylight — APPROVED** [review]
I've re-tested this on a live environment, and have not found any detriment to functionality. LGTM!

**Pull request merged**